src/presets/index.ts

```diff
@@ -7,7 +7,7 @@ export * from './deno'
 export * from './digital-ocean'
 export * from './firebase'
 export * from './heroku'
-export * from './layer0'
+export * from './edgio'
```

Comment:
This is a breaking change. I hope it is fine for your current customers...

Reply:
Maybe it'd be good to keep it as an alias?

Reply:
Yes